### PR TITLE
Fix jaws db url to get app to load in Heroku

### PR DESCRIPTION
**Pull request by vladgolitsyn**
## Description
Updates db environment variable to allow Heroku to properly load the app

## Ticket being worked on
[DEV-1234](link to jira)

## Acceptance criteria
- [ ] Heroku should successfully load the app and display `home` and `events` pages

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "dialect": "mysql"
   },
   "production": {
-    "use_env_variable": "JAWS_DB_URL",
+    "use_env_variable": "JAWSDB_URL",
```
Comment (Rahul-Vig) [inline config/config.json:17]: :shark:

File: models/index.js [hunk 2]
```diff
@@ -1,27 +1,33 @@
-'use strict';
+"use strict";
```
Comment (Rahul-Vig) [inline models/index.js:1]: is this just prettier or did we change anything in this file?

File: routes/eventRoutes.js [hunk 3]
```diff
@@ -48,44 +43,40 @@ app.get("/events/:category", function (req, res) {
...
-        date: req.body.date,
-        location: req.body.location,
-        description: req.body.description,
-        category: req.body.category, //FYI this is not in events.js file
-        participants: req.body.participants
-    })
-        .then(function (dbPost) {
-            res.json(dbPost);
-        });
+app.post("/events", function(req, res) {
```
Comment (Rahul-Vig) [inline routes/eventRoutes.js:46]: is this preier again?

File: server.js [hunk 4]
```diff
@@ -18,11 +18,12 @@ app.set("view engine", "handlebars");
...
-db.sequelize.sync({}).then(function() {
+db.sequelize.sync().then(function() {
   app.listen(PORT, function() {
     console.log(
       "Climatize Web App Server listening on: http://localhost:" + PORT
     );
   });
 });
+
+// db.sequelize.sync({}).then(function() {});
```
Comment (Rahul-Vig) [inline server.js]: we should delete this

**Review by Rahul-Vig — COMMENTED** [review]
LGTM :+1:  I just made some minor comments and had one change request :100:

File: models/index.js (hunk 2 above)
Comment (vladgolitsyn) [reply models/index.js:1]: yes this was just prettier, I din't change anything here.

File: routes/eventRoutes.js (hunk 3 above)
Comment (vladgolitsyn) [reply routes/eventRoutes.js:46]: yep

File: server.js (hunk 4 above)
Comment (vladgolitsyn) [reply server.js]: ooh good catch, let me delete that

**Review by Rahul-Vig — APPROVED** [review]
:shark:

**Pull request merged**